jcli/src/jcli_app/rest/v0/diagnostic/mod.rs

```diff
@@ -11,17 +11,19 @@ pub enum Diagnostic {
         addr: HostAddr,
         #[structopt(flatten)]
         debug: DebugFlag,
+        #[structopt(flatten)]
+        tls: TlsCert,
```

Comment:
These options are getting a bit repetitive, maybe there should be a struct for them that will be flattened into commands' options.

Reply:
Actually, I thought about making a single structure out of `addr, `debug` and `tls`, that will have a method to build a client perform a request.